src/main/resources/db/migration/V12__imrt_ddl.sql

```diff
@@ -0,0 +1,5 @@
+/****************************************************
+ * Add item author field
+ ****************************************************/
+
+alter table item add column item_author character varying NOT NULL DEFAULT '';
```

Comment:
Minor nit-pick: Since this implies ownership/authorship, should this be `item_authored_by`?  Or maybe just `author`, since we know it's in the `item` table?

Reply:
It is called itemAuthor in the Item Data Dictionary and the item.json which is why I went with this value.  it is a user entered field so it isn't who "created" the item necessarily.